.github/workflows/rust-ci.yaml

```diff
@@ -148,8 +149,8 @@ jobs:
           cat ${GITHUB_ENV}
       - name: Cargo Test
         run: |
-          nix develop -c \
-            cargo test --all --all-features --all-targets $EXCLUDES
+          SOUNDS_DIR=$(pwd)/ui/sound/assets nix develop -c \
```

Comment:
i don't like setting this variable here but i didn't find a better approach :/

Reply:
lets use a config struct rather than this global env var